firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionHelpers/TrackingProtectionConnectionDetailsView.swift

```diff
@@ -131,4 +131,8 @@ final class TrackingProtectionConnectionDetailsView: UIView {
     func setupAccessibilityIdentifiers(foxImageA11yId: String) {
         foxStatusImage.accessibilityIdentifier = foxImageA11yId
     }
+
+    func applyTheme(theme: Theme) {
+        backgroundColor = theme.type == .privateMode ? theme.colors.layerAccentPrivateNonOpaque : theme.colors.layer2
```

Comment:
My comment from https://github.com/mozilla-mobile/firefox-ios/pull/22899#discussion_r1831507878 applies here as well. 

"I believe this goes against the purpose of having a theming system in that we have to check the type of theme instead of setting one color, if I am understanding this correctly.

Happy to open it to the team, but maybe this is something we need to discuss with the designer?"

Reply:
updated with the correct colors